js/src/features/human-resouce/components/Absence/Abcence.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback } from "react";
+import { List, Tag, Tooltip } from "antd";
+import {
+  CheckCircleOutlined,
+  CloseCircleOutlined,
+  EditOutlined,
+} from "@ant-design/icons";
+import { Absence as AbsenceModel } from "HumanResourceModels";
```

Comment:
How about don't rename Absence model and specify Absence component? In the future this will help to avoid problems with name collision.

Reply:
What do you mean 'specify'? Rename to AbsenceComponent ?

Reply:
It means that Absence can be `AbsenceListItem` or `AbsanceDetails` or something else.

Reply:
Component is not the best way to show affiliation to UI. There are more expressive words.

Reply:
You renamed component, but left AbsenceModel